sub-packages/bionemo-geneformer/src/bionemo/geneformer/scripts/train_geneformer.py

```diff
@@ -192,8 +192,8 @@ def main(
         gradient_as_bucket_view=True,
         # FIXME there are intermittent errors with async checkpoint saving.
         #  see https://wandb.ai/clara-discovery/geneformer_bionemo2_goodslurm/runs/uAFi7DzI/logs
```

Comment:
Remove this comment as well.

Reply:
Maybe add a warning about how async save will fail if checkpoint names are not unique enough.